CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1.0 February 1, 2024
+
+- **Breaking Change** - a `CheckoutCompletedEvent` parameter has been added to the `onCheckoutCompleted` callback, providing clients with more information about the checkout that just completed. 
```

Comment:
Does this have to be a breaking change?

Reply:
We potentially could have two overloads of onCheckoutCompleted

Reply:
Overload + mark the old as deprecated?

Reply:
yeah, sounds reasonable 👍